### PR TITLE
Fixes #2675 Module names in the Modules debugger window are unqualified

### LINKED ISSUE #2675: Module names in the Modules debugger window are unqualified
We use unqualified names in the "Name" column of the Modules tool window - e.g. `abc`, `collections.abc` and `encodings.abc` are all reported as three different entries all named `abc`. You can distinguish them by looking at the module path, but we should really be using properly qualified names as well.

This is particularly bad with respect to the numerous `__init__` entries.

### PR DESCRIPTION
**Pull request by zooba**
Fixes #2675 Module names in the Modules debugger window are unqualified
Includes full module name when we have it
Correctly marks non-user modules
Fixes ptvsd directory not being excluded

### CODE REVIEW

File: Python/Product/Debugger/Debugger/DebugEngine/AD7Module.cs [hunk 1]
```diff
@@ -110,7 +120,7 @@ class AD7Module : IDebugModule2, IDebugModule3 {
         // the sample debug engine does not support JustMyCode and therefore all modules
         // are considered "My Code"
         int IDebugModule3.IsUserCode(out int pfUser) {
-            pfUser = 1;
+            pfUser = DebuggedModule.IsUserCode ? 1 : 0;
```
Comment (int19h) [inline Python/Product/Debugger/Debugger/DebugEngine/AD7Module.cs]: Can you also get rid of the comment above, while you're at it?

File: Python/Product/Debugger/Debugger/LegacyDebuggerProtocol.cs [hunk 2]
```diff
@@ -405,6 +406,10 @@ public sealed class ModuleLoadEvent : Event {
 
             public int moduleId;
             public string moduleFileName;
+            [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+            public string moduleName;
+            [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
+            public int isStdLib;
```
Comment (int19h) [inline Python/Product/Debugger/Debugger/LegacyDebuggerProtocol.cs]: bool?

Comment (zooba) [reply Python/Product/Debugger/Debugger/LegacyDebuggerProtocol.cs]: Kept blowing up in deserialization for some reason I couldn't figure out. I think the JSON serialization is more forgiving for missing/null int than bool.

Comment (zooba) [reply Python/Product/Debugger/Debugger/LegacyDebuggerProtocol.cs]: Tried changing it back to `bool` and it worked, so I guess I messed something up last time.

**Pull request merged**